### ISSUE TITLE
运行MOSEI数据集时报错问题

### ISSUE THREAD

**Issue by GOSkype** [kiva12138/CubeMLP#10]
当我运行了python Train.py --dataset mosei_SDK --batch_size 64 --features_compose_t mean --features_compose_k cat --d_hiddens 20-3-128=10-2-64=5-2-32  --d_outs 20-3-128=10-2-64=5-1-32 --res_project 1-1-1--bias --ln_first  --dropout_mlp 0.1-0.1-0.1 --dropout 0.1-0.1-0.1-0.1 --bert_freeze part --bert_lr_rate 0.01  --learning_rate 4e-3 --video facet42 
出现了报错，请问如何解决
![1](https://github.com/kiva12138/CubeMLP/assets/39805002/50cbb3d1-fdc5-49c1-9164-225fbcd62e03)

**Comment by kiva12138** [comment]
抱歉，由于MOSEI提供的数据集中，视频的facet42特征为35维，而默认的视频特征维度为47，因此需要在运行时加上命令_--d_v 35_. 非常抱歉没有详细阐述

**Comment by GOSkype** [comment]
> 抱歉，由于MOSEI提供的数据集中，视频的facet42特征为35维，而默认的视频特征维度为47，因此需要在运行时加上命令_--d_v 35_. 非常抱歉没有详细阐述

哈哈哈，好的，谢谢你的回答
